internal/pkg/database/main_test.go

```diff
@@ -1,18 +1,32 @@
 package database
 
 import (
+	"log"
 	"os"
 
-	"github.com/jmoiron/sqlx"
 	"testing"
+
+	"github.com/jmoiron/sqlx"
 )
 
 var db *sqlx.DB
+var closedDb *sqlx.DB
 
 // Initializes the database for the tests run in this package
 func TestMain(m *testing.M) {
-	var td func()
+	var td, closedTd func()
+
 	td, db = StartTestPostgreSQL()
+	closedTd, closedDb = StartTestPostgreSQL()
+	if err := closedDb.Close(); err != nil {
+		log.Fatalf("unable to close database: %s", err)
+	}
+	closedTd()
```

Comment:
Not sure what this does? Could use a comment.

Reply:
Added at the declaration of the variable.